client/src/components/to-do-list/Category.jsx

```diff
@@ -1,7 +1,8 @@
-import React, {useState, useEffect, useCallback} from 'react';
+import React, {useState, useEffect, useCallback, Suspense} from 'react';
```

Comment:
Planning to lazy load something here later?  Don't see any usage of react.suspense here.

Reply:
I was going to use lazy load for TaskOptionsModal import but it wasn't working so I just did a regular import and forgot to take out Suspense. Will remove.